contract-ui/tabs/listings/components/table.tsx

```diff
@@ -193,28 +221,6 @@ export const ListingsTable: React.FC<ListingsTableProps> = ({ contract }) => {
                 </Tr>
               );
             })}
-            {getAllQueryResult.isPreviousData && (
```

Comment:
can we instead keep this but also check if the filter didn't change? to do that use a `usePrevious()` (we have it in other places) and check it here, too before showing the overlay